fractal-puppy/Engine
# Image Mask can go vertically and horizontally

Now you can choose if the mask applied to an image is vertical or horizontal
To test this:
- Create an image in the hierarchy (right click -> ui -> image). 
- Mark in the panel "Has Mask" and "is horizontal mask".
- Move the value below the checks and check if the image reduces and increases horizontally

Note: You can't see the image if you don't create a camera for the viewport game.
